CHANGES.rst

```diff
@@ -10,7 +11,7 @@
 2.0.0a3 (2015-12-03)
 ====================
 
-- [NEW] Added API documentation hosted on readthedocs.org.
+- [NEW] Added API documentation hosted on readthedocs.com.
```

Comment:
I'm equally confused as to why I got it in my head that this needed to be changed.  I'll have to chalk this up to the fog of working with the documentation for so long.  Anyway, I changed this back to .org in d807820.